### PR TITLE
extract benchmark-report component

### PR DESCRIPTION
**Pull request by GavinJoyce**
This adds back the HTML report and simplifies the index controller by extracting the report behavior to a component. 

The 'Submit my Results' function is broken on master, I'll fix this weekend (https://github.com/eviltrout/ember-performance/issues/69)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,7 +18,6 @@
     "node": ">= 0.10.0"
   },
   "author": "Robin Ward",
-  "license": "MIT",
```
Comment (GavinJoyce) [inline package.json]: oops, adding this back in now

Comment (GavinJoyce) [inline package.json]: ah, it was in there twice. it's on line 5

**Comment by eviltrout** [comment]
Fantastic work. I'm going to give you that commit bit now :)

**Comment by GavinJoyce** [comment]
nice one, thanks

**Pull request merged**